code/src/App.js

```diff
@@ -1,9 +1,31 @@
-import React from 'react'
+import React from 'react';
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
+
+import Header from 'components/Header';
+
+import Trending from 'pages/Trending';
+import Movie from 'pages/DetailsMovie';
+import Collection from 'pages/DetailsCollection';
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <BrowserRouter>
+      <Header />
+      <Switch>
+        <Route exact path="/">
+          <Trending />
+        </Route>
+        <Route path="/movies/:id">
+          <Movie />
+        </Route>
+        <Route path="/collections/:id">
+          <Collection />
+        </Route>
+        {/* If user tries to go to a route that is non-existent we redirect to main page */}
+        <Route>
+          <Redirect to="/" />
```

Comment:
Very good use of Route that you take usage of different information from the API. Super impressed🥇